jest.config.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     '/node_modules/',
     'app/src/ipc.js',
     'app/src/ledger.js',
+    'app/src/modules/process.js',
```

Comment:
I miss some comment explaining why the file coverage is ignored. 
```suggestion
    'app/src/modules/process.js', // this file is ignored because it's just a simple wrapper created to allow mocking `process` in unit tests. 
```
Maybe it would be even more clear if the the explanation and `istanbul ignore` would be inside the `process.js` file

Reply:
done